providers/server_monitor.rb

```diff
@@ -98,7 +98,7 @@ def update_newrelic_alert_policy_linux(alert_policy_id)
     end
 
     only_if do
-      node['newrelic']['api_key'].!empty?
+      !node['newrelic']['api_key'].empty?
```

Comment:
One can use `not_if` block instead and drop the `!`. Would be much cleaner this way.